pkgs/top-level/all-packages.nix

```diff
@@ -6040,9 +6040,13 @@ in
   # with different versions we pin Go for all versions.
   nomad_0_11 = callPackage ../applications/networking/cluster/nomad/0.11.nix {
     buildGoPackage = buildGo114Package;
+    inherit (linuxPackages) nvidia_x11;
+    nvidiaGpuSupport = config.cudaSupport or (!stdenv.isLinux);
```

Comment:
I'm not sure if we want to co-opt `config.cudaSupport` here, since this has nothing to do with cuda. I used it because that seems to be the convention for enabling NVidia GPU-related features.

Happy to do something else here though.

Reply:
That seems fine by me 👍

Reply:
Cool. Thinking about it some more, I think another argument in favor of using `cudaSupport` is that if someone sets that to true, but then cannot use nomad's GPU device plugin that'd be real surprising.